internal/commands/all/purge.go

```diff
@@ -0,0 +1,65 @@
+package all
+
+import (
+	"fmt"
+
+	"github.com/UpCloudLtd/progress/messages"
+	"github.com/UpCloudLtd/upcloud-cli/v3/internal/commands"
+	"github.com/UpCloudLtd/upcloud-cli/v3/internal/output"
+
+	"github.com/spf13/pflag"
+)
+
+// PurgeCommand creates the "all purge" command
+func PurgeCommand() commands.Command {
+	return &purgeCommand{
+		BaseCommand: commands.New(
+			"purge",
+			"Delete UpCloud resources within the current account",
+			"upctl all purge",
+			"upctl all purge --name \"*tf-acc-test*-\"",
+		),
+	}
+}
+
+type purgeCommand struct {
+	*commands.BaseCommand
+	name string
+}
+
+func (c *purgeCommand) InitCommand() {
+	flags := &pflag.FlagSet{}
+	flags.StringVar(&c.name, "name", "", "Only delete resources matching the given name.")
+	c.AddFlags(flags)
+}
+
+// ExecuteWithoutArguments implements commands.NoArgumentCommand
+func (c *purgeCommand) ExecuteWithoutArguments(exec commands.Executor) (output.Output, error) {
+	msg := "Deleting all resources"
```

Comment:
Do we need confirmation prompt? I think this would mostly executed in CI cleanups with `-y` anyway, so maybe not 🤔

Reply:
Maybe not needed.